src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -87,7 +88,8 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		public connectionProfile: IConnectionProfile | undefined,
 		@ILogService private readonly logService: ILogService,
 		@INotificationService private readonly notificationService: INotificationService,
-		@ITelemetryService private readonly telemetryService: ITelemetryService
+		@ITelemetryService private readonly telemetryService: ITelemetryService,
+		@IAdsTelemetryService private readonly _adstelemetryService: IAdsTelemetryService
```

Comment:
For someone not super familiar with the differences between the ADS and VS Code telemetry systems, why can't we just use one?

Reply:
There should be only one. @rajeshka would you please update the other telemetry call to also use AdsTelemetryService and then delete the reference to ITelemetryService? It should end up like this :

```
this.adsTelemetryService.sendMetricsEvent( { markdownRenderingElapsedMs: markdownRenderingTime }, TelemetryView.Notebook);
```

and then delete the old NotebookMarkdownRendered key.

@chlafreniere This will mean that if wanted to compare this metric between old events and this new one we'd have to do a little additional processing. But given that we don't seem to be using this currently anywhere I think that's fine - you OK with that?

Reply:
@chlafreniere  - Agreed. Was thinking about it but wanted for PR comments.
@Charles-Gagnon  - On it.

Reply:
Done